assertion/function/assertiontree/trusted_func.go

```diff
@@ -145,7 +145,7 @@ var negatedSelfExpr action = func(call *ast.CallExpr, argIndex int, _ *analysis.
 	}
 }
 
-var nonnilProducer action = func(call *ast.CallExpr, argIndex int, _ *analysis.Pass) any {
+var nonnilProducer action = func(call *ast.CallExpr, _ int, _ *analysis.Pass) any {
```

Comment:
(unrelated to this PR, but needed to add this lint fix to make the linter happy)